src/components/MonitorLog/AdditionalSpeciesTab.jsx

```diff
@@ -25,12 +28,14 @@ import {
 import PropTypes from 'prop-types';
 import React, { useEffect, useState, useMemo } from 'react';
 import { useFormContext } from 'react-hook-form';
+import { FiExternalLink } from 'react-icons/fi';
+import { Link } from 'react-router-dom';
 import AddSpeciesModal from './AddSpeciesModal';
 import EditSpeciesModal from './EditSpeciesModal';
 
 const FORM_PREFIX = 'additionalSpecies.';
 
-const AdditionalSpeciesTab = ({ showHeader, isDisabled, species }) => {
+const AdditionalSpeciesTab = ({ showHeader, isDisabled, isTemplate, species }) => {
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'species' is missing in props validation